metadata.rb

```diff
@@ -9,7 +9,7 @@
 recipe 'passenger_apache2::default', 'Installs Passenger as an Apache module'
 recipe 'passenger_apache2::mod_rails', 'Enables Apache module configuration for passenger module'
 
-depends 'apache2', '>= 2.0'
+depends 'apache2', '~> 7.1'
```

Comment:
You probably want to just make this >= 7.1. Aggressive version constraints have been super problematic historically in community cookbooks.

Reply:
Ok. Yeah, I don't think you can currently win with this -> in our team I think we've had issues both with cookbook version constraints being too lax and too aggressive. I can see that it's more flexible to make them less aggressive here & pass on the responsibility to downstream users to pin more aggressively as/when required, though, so I've changed it as you suggest.